### PR TITLE
Update DMSScript-options.md

### PR DESCRIPTION
**Pull request by dmainha**
For an easier decision that reference explains the flags in more detail

### CODE REVIEW

**Comment by MariekeGO** [comment]
@dmainha, could you add your full name to your GitHub profile so we can assign DevOps points if you are a member of the DevOps Program?

**Comment by dmainha** [comment]
hi @MariekeGO not yet part of the program. will look into that.

File: develop/schemadoc/Automation/DMSScript-options.md
Comment (MariekeGO) [inline develop/schemadoc/Automation/DMSScript-options.md:1]: As the page that is linked to only contains info for some of these flags, I'm wondering if it wouldn't be better if we simply explained all flags here. @PedroDebevere, is this something you could take care of?

Comment (PedroDebevere) [reply develop/schemadoc/Automation/DMSScript-options.md:1]: @MariekeGO, provided more info about each option.

**Review by MariekeGO — APPROVED** [review]

File: develop/schemadoc/Automation/DMSScript-options.md
Comment (dmainha) [reply develop/schemadoc/Automation/DMSScript-options.md:1]: It looks better now. Thanks

**Pull request merged**